.vscode/settings.json

```diff
@@ -26,5 +26,9 @@
   "tslint.alwaysShowRuleFailuresAsWarnings": true,
   "css.validate": false,
   "scss.validate": false,
-  "python.formatting.provider": "yapf"
+  "python.formatting.provider": "yapf",
+  "[json]": {
```

Comment:
can you revert that? We use .editorConfig for those settings and json is set to 2 space on purpose as npm will use 2 space for package.json